### PR TITLE
Support on all device

### PR DESCRIPTION
**Pull request by V0lk7**
- add the power to move with the mouse on the game
- solved the issue in the tournament with the reverse paddle and username
- clean all js code of console logs
- clean a little the html to be more consistent
- add the round to the winrate to have only two digit after the VIRGULE

### CODE REVIEW

File: Django_data/apps/Home/views.py
Comment (ji2myji2my) [inline Django_data/apps/Home/views.py:1]: <3

File: Django_data/static/js/Game/Parameters_Events.js
Comment (ji2myji2my) [inline Django_data/static/js/Game/Parameters_Events.js:1]: <3

File: Django_data/static/js/User/Settings_Callback.js
Comment (ji2myji2my) [inline Django_data/static/js/User/Settings_Callback.js:1]: <3

**Review by ji2myji2my — APPROVED** [review]
Master <3

File: Django_data/static/js/User/Settings_Callback.js
Comment (V0lk7) [reply Django_data/static/js/User/Settings_Callback.js:1]: <3

**Pull request merged**